src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+abstract class Figure implements Area, Drawable {
```

Comment:
Don't use verbs for class/interface names. In this case, 'Area' should be named in a way that represents what it is or does, such as 'AreaCalculatable' or 'AreaProvider' to adhere to proper naming conventions.